### PR TITLE
Adds tests, small improvements in display of tables and plots, fix of KLD issue

### PR DESCRIPTION
**Pull request by juliuspf**
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/1026
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/993
partly: https://github.com/jasp-stats/INTERNAL-jasp/issues/1027

### CODE REVIEW

**Comment by TimKDJ** [comment]
Is this PR up-to-date with https://github.com/jasp-stats/jasp-desktop/pull/4249?

**Comment by juliuspf** [comment]
What do you mean by that?

**Comment by TimKDJ** [comment]
Are all the changes you made in the jasp-desktop pull request (e.g., review comment changes), included in this PR?

**Comment by juliuspf** [comment]
Ah, ja! The R-files are exactly the same

**Comment by juliuspf** [comment]
Just the tests are new here

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
+
+os: osx
+language: r
+r: 3.6.3
+r_github_packages: jasp-stats/jaspTools
```
Comment (TimKDJ) [inline .travis.yml:6]: Could you use https://github.com/jasp-stats/jasp-desktop/blob/stable/JASP-Tests/README.md#travisyml?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
-Date: 2020-06-17
+Version: 0.1.0
+Date: 2020-08-19
 Author: Julius M. Pfadt, Don van den Bergh & Eric-Jan Wagenmakers
-Website: https://github.com/juliuspf/Reliability
+Website: https://github.com/jasp-stats/Reliability
 Maintainer: Julius M. Pfadt <julius.pfadt@gmail.com>
-Description: TODO
-License: GPL (>= 2)
-Imports: Rdpack, Bayesrel, Rcsdp (>= 0.1.57)
```
Comment (TimKDJ) [inline DESCRIPTION:11]: These don't look like changes we necessarily want. Are they made on purpose?

File: tests/testthat/test.csv [hunk 3]
```diff
@@ -0,0 +1,101 @@
+,contNormal,contGamma,contBinom,contExpon,contWide,contNarrow,contOutlier,contcor1,contcor2,facGender,facExperim,facFive,facFifty,facOutlier,debString,debMiss1,debMiss30,debMiss80,debMiss99,debBinMiss20,debNaN,debNaN10,debInf,debCollin1,debCollin2,debCollin3,debEqual1,debEqual2,debSame,unicode
```
Comment (TimKDJ) [inline tests/testthat/test.csv:1]: This file is now shipped with `jaspTools` so you don't need to include it

**Review by TimKDJ — CHANGES_REQUESTED** [review]
Looks good, just minor changes required

**Review by TimKDJ — APPROVED** [review]

**Pull request merged**